nx-X11/programs/Xserver/hw/nxagent/NXcomposite.h

```diff
@@ -1,86 +0,0 @@
-#ifndef NXAGENT_UPGRADE
```

Comment:
Why? We previously built nxagent with NXAGENT_UPGRADE being defined, so we can drop all passages that would be built with NXAGENT_UPGRADE being undefined, which is exactly the case for the above quoted code line.